### PR TITLE
Eslint Addition

### PR DESCRIPTION
**Pull request by bibekluitel**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "env": {
+    "node": true,
+    "es6": true,
```
[the remark this answers is not in the document]
Comment (bibekluitel) [reply .eslintrc.json]: Ya i forgot that  part.

[the remark this answers is not in the document]
Comment (bibekluitel) [reply .eslintrc.json]: we cannot use const and let  too  after this.

[the remark this answers is not in the document]
Comment (arshadkazmi42) [reply .eslintrc.json]: So what's your call, I think we should keep es6 only.

[the remark this answers is not in the document]
Comment (bibekluitel) [reply .eslintrc.json]: ya .. scoping of variables creats a huge problems while working. let and const solves those scoping problems.

[the remark this answers is not in the document]
Comment (arshadkazmi42) [reply .eslintrc.json]: cool.

**Pull request merged**